backend/vcdat/app.py

```diff
@@ -78,5 +78,6 @@ def load_variables_from_file():
     f = cdms2.open(file_path)
     return json.dumps({'variables': f.listvariables()})
 
+
```

Comment:
Sorry, I was trying to figure out if the "app.py missing 81,82" on the coverage report was referring to the lines that were not being covered. So I just added a blank line and ran the coverage report again to see if the number was referring to the line by changing to "app.py missing 82,83". I forgot to undo it